apps/vr-tests-react-components/src/stories/SearchBox.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { SearchBox } from '@fluentui/react-search';
+import { ArrowEnterLeftRegular, MicRegular, PersonRegular } from '@fluentui/react-icons';
+import { TestWrapperDecorator } from '../utilities/TestWrapperDecorator';
+import { FluentProvider } from '@fluentui/react-provider';
+
+// Focused & unfocused stories
```

Comment:
A couple others I'd suggest are:
- hover state
- focusing the dismiss button (this would need a separate storiesOf + set of steps)
- disabled state
- disabled + hover